src/util/piped_process.cpp

```diff
@@ -193,13 +206,7 @@ piped_processt::piped_processt(const std::vector<std::string> commandvec)
   start_info.hStdOutput = child_std_OUT_Wr;
   start_info.hStdInput = child_std_IN_Rd;
   start_info.dwFlags |= STARTF_USESTDHANDLES;
-  // Unpack the command into a single string for Windows API
```

Comment:
:pick: I suggest reusing this comment as doxygen just above the definition of the new function.